src/async_wrap-inl.h

```diff
@@ -26,10 +26,21 @@
 
 #include "async_wrap.h"
 #include "base_object-inl.h"
+#include "node_errors.h"
 #include "node_internals.h"
+#include "v8.h"
```

Comment:
It doesn’t really matter, but the other files already have to include v8.h to function. I don’t think you need either header that is being added here

Reply:
@addaleax, thanks. Yes, it works without v8.h, so removed it.